src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -58,20 +60,21 @@ OtpUser preCreateHook(OtpUser user, Request req) {
 
     @Override
     protected void buildEndpoint(ApiEndpoint baseEndpoint) {
-        LOG.info("Registering path {}.", ROOT_ROUTE + "verify");
+        LOG.info("Registering path {}/{}.", ROOT_ROUTE, VERIFY_PATH);
 
         // Add the api key route BEFORE the regular CRUD methods
         ApiEndpoint modifiedEndpoint = baseEndpoint
-            .get(path(ROOT_ROUTE + ID_PATH + VERIFY_ROUTE)
+            .get(path(ROOT_ROUTE + String.format(VERIFY_ROUTE_TEMPLATE, ID_PARAM, VERIFY_PATH, PHONE_PARAM))
                     .withDescription("Request an SMS verification to be sent to an OtpUser's phone number.")
-                    .withPathParam().withName(ID_PARAM).withDescription("The id of the OtpUser.").and()
+                    .withPathParam().withName(ID_PARAM).withRequired(true).withDescription("The id of the OtpUser.").and()
+                    .withPathParam().withName(PHONE_PARAM).withRequired(true).withDescription("The phone number to validate, in E.164 format.").and()
```

Comment:
While precise, this isn't super helpful. It would be good to add an example to this description (e.g., `+15555551234`).

Reply:
This item is unresolved.

Reply:
Fixed. in https://github.com/ibi-group/otp-middleware/pull/83/commits/e1e2cd22c54f645d30c6662dbf1e8e366797c6ce

Reply:
Sorry, I didn't mean to remove that it should be E.164 (which is nice because it is precise), just that we should add the example (for folks who don't immediately know what that means).